### PR TITLE
Front page nas optimization & readme update

### PR DESCRIPTION
**Pull request by alneberg**
I tested this against tools-dev, not sure if anything else is required before it is considered safe?

The new assumption (< 200 NAS:es) is perhaps not super pretty but it only affects the front page of genomics status as the logic is specific to that handler.

This is a quick fix, I didn't want to restructure it more than necessary in order to get it to load faster.

### CODE REVIEW

**Comment by ewels** [comment]
Nice! To be fair, you can probably drop this down to 20. Any suggestion of how much of a difference this makes?

File: README.md [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 
-Status interfaces with StatusDB; the CouchDB database instance we're using at SciLifeLab to store metadata in 
+Status interfaces with StatusDB; the CouchDB database instance we're using at SciLifeLab to store metadata in
 various forms. Document specifications for StatusDB are available in the internal wiki. Documentation about CouchDB can be found [here](http://guide.couchdb.org/).
 
 ## Installing and running genomics status
 
 **NOTE**: These steps assume that:
 * you're either running a python virtualenv or you do have root permissions.
+* you're running python version 2.X
```
Comment (remiolsen) [inline README.md]: I think we should assume version 2.7. Woe to those who run python 2.6.

**Comment by alneberg** [comment]
> To be fair, you can probably drop this down to 20. Any suggestion of how much of a difference this makes?

Yeah, I wanted to be on the safe side. The difference from 200 to 20 is probably quite tiny.

File: README.md (hunk 1 above)
Comment (alneberg) [reply README.md]: Fine by me. I just noticed that python 3 was a nono.

**Comment by ewels** [comment]
Nice! 👌 

Please deploy on production now :)

**Comment by alneberg** [comment]
> Please deploy on production now :)

I hope I'm not supposed to do that? 😨

**Comment by ewels** [comment]
Yeah please :) You can ask @remiolsen or @senthil10 or someone for help.

**Pull request merged**